docs/doxygen/ngraph_py_api.xml

```diff
@@ -0,0 +1,200 @@
+<doxygenlayout version="1.0">
```

Comment:
Do we really need to create different files with the same context for each API reference?

Reply:
Yes, this the way how it works right now. For future releases, we're going to optimize the structure